packages/rainbowkit/src/components/AsyncImage/AsyncImage.tsx

```diff
@@ -62,12 +62,12 @@ export function AsyncImage({
               'src': src,
             }
           : {
-              backgroundSize: 'cover',
+              backgroundSize: 'contain',
             })}
         height="full"
         position="absolute"
         style={{
-          touchCallout: 'none',
+          WebkitTouchCallout: 'none',
```

Comment:
Weird that CI/CD didn't catch this issue, but it somehow catches in this PR. However the `touchCallout` doesn't exist and we have to use `WebkitTouchCallout`.